.gitignore

```diff
@@ -2,6 +2,7 @@
 .pypirc
 ~*
 tmp*
+sample_data/
```

Comment:
Could you rename to `tmp_sample_data` - that way it'll be ignored already, and we won't need anything added to .gitignore